x-pack/plugins/observability_solution/dataset_quality/public/services/telemetry/telemetry_client.ts

```diff
@@ -55,4 +55,10 @@ export class TelemetryClient implements ITelemetryClient {
       tracking_id: this.datasetDetailsTrackingId,
     });
   };
+
+  public trackDatasetDetailsBreakdownFieldChanged = (eventProps: DatasetDetailsEbtProps) => {
+    this.analytics.reportEvent(DatasetQualityTelemetryEventTypes.BREAKDOWN_FIELD_CHANGED, {
+      ...eventProps,
```

Comment:
We also need to pass `tracking_id` (like above events), this will help us analyze how many times on average a user changes the breakdown field before going to a particular destination.